ofrak_core/Dockerstub

```diff
@@ -14,7 +14,6 @@ RUN apt-get -y update && \
       mtd-utils \
       pigz \
       zip \
-      qemu \
```

Comment:
Are we removing qemu on purpose?

Reply:
This package was not available in the Debian 12 repos, so I thought I would remove it as it isn't used by anything. In the repos qemu is broken out in multiple packages for system or user mode, for different architectures, etc. Do we need to install it? What's the use case?